vercel/resource_project_test.go

```diff
@@ -833,6 +862,104 @@ resource "vercel_project" "test" {
 `, projectSuffix, teamID)
 }
 
+func testAccProjectConfigWithoutEnableAffectedSet(projectSuffix, teamID string) string {
+	return fmt.Sprintf(`
+resource "vercel_project" "test" {
+  name = "test-acc-project-%s"
+  %s
+  build_command = "npm run build"
```

Comment:
You can remove most of these settings. I think name and the %s for team config are the only required things here. The rest add a lot of noise to the test

Reply:
done!